source/getting-started/tutorials/timing-manager/index.md

```diff
@@ -0,0 +1,309 @@
+- **Complexity:** 3 / 5
+- **Estimated Time:** 40 min
+
+
+## Tutorial Requirements
+
+1. Working AMDC Hardware
+2. Completion of the [Hardware Commands](/getting-started/tutorials/hw-commands/index.md) tutorial
+3. Completion of the [Voltage Source Inverter](/getting-started/tutorials/vsi/index.md) tutorial
+4. Completion of the [Profiling Tasks](/getting-started/tutorials/profiling-tasks/index.md) tutorial
+
+This tutorial expands on the code created in the [Voltage Source Inverter](/getting-started/tutorials/vsi/index.md) tutorial, and uses commands created in the [Profiling Tasks](/getting-started/tutorials/profiling-tasks/index.md) tutorial. Both must be completed before this tutorial.
+
+## Introduction
+
+In motor control applications, it is important for the timing of both sensor data acquisition and control task execution to remain consistent. In its default configuration, the AMDC may allow some jitter (timing incosistency) between sensor measurement time and control loop timing. This tutorial shows how the AMDC's [Timing Manager](/firmware/arch/timing-manager.md) peripheral can be configured to eliminate this jitter.
+
+This tutorial provides code that allows users to experiment with the configuration of the AMDC [Timing Manager](/firmware/arch/timing-manager.md). The effects of this experimentation will be observed through the lens of the previous [Voltage Source Inverter](/getting-started/tutorials/vsi/index.md) tutorial.
+
+## Scheduling and Synchronizing
+
+Through the [Timing Manager](/firmware/arch/timing-manager.md), the AMDC is capable of synchronizing sensor data collection and task execution to the PWM carrier wave. This is desirable to eliminate electromagnetic noise in motor drives (for example, switching harmonics in the phase current measurements) because the inverter does not switch at the peak and trough of the PWM carrier.
+
+Additionally, updates to the PWM duty ratio can be precisely synced with the timing manager, and are by default.
+
+### Timing Manager Modes
+
+Every `X` PWM periods (where the user specifies `X` using the function `timing_manager_set_ratio()`), the AMDC will collect data from the sensors. By default, the [Timing Manager](/firmware/arch/timing-manager.md) is set to `Legacy Mode`, in which the AMDC OS will run tasks independent of whether the sensors have completed data collection.
+
+Alternatively, when configured in `Post-Sensor Mode`, the [Timing Manager](/firmware/arch/timing-manager.md) will instruct the AMDC OS to wait to execute tasks until the sensor data collection is complete. With proper configuration, this can be used to ensure the following:
+
+1. the task has new sensor data and
+2. the time elapsed between when the sensor data is acquired and when the task is run is consistent and jitter-free.
+3. the pwm duty ratio is updated at a consistent rate (every X pwm cycles)
+
+### Configuring the Timing Manager
+
+There are multiple factors that affect when and how fast control tasks run:
```

Comment:
The purpose of this subsection should be to explain 
- what actions the timing manager synchronizes (based on PWM carrier: 1) start signal for all sensors to start acquiring data and 2) OS interrupt for the next task to run)
- The fact that there is not one special task here. All tasks are treated equal.
- Tell us that the OS will not allow any task to run more than once per timing manager event. 
- The rules that must be followed (inequlities). Clean 'em up a bit. `Sensor Sample Acquisition Time`, `Task Run Time`, `TASKNAME_UPDATES_PER_SEC`, `PWM_FREQUENCY`, `EVENT_RATIO`
    - Improve the typesetting on what each of these control